.circleci/config.yml

```diff
@@ -49,9 +51,6 @@ jobs:
             - build/libs
             - build.gradle.kts
             - app_version.txt
-      - slack/notify:
-          event: fail
-          template: basic_fail_1
```

Comment:
Don't see the point in this notification - devs will notice if their build fails pretty quickly